components/dashboard/src/start/StartPage.tsx

```diff
@@ -105,6 +107,17 @@ export function StartPage(props: StartPageProps) {
                 )}
                 {error && <StartError error={error} />}
                 {props.children}
+                {props.showLatestIdeWarning && (
+                    <InfoBox className="mt-4">
+                        You are using the latest editor version.{" "}
```

Comment:
**fyi**: See https://github.com/gitpod-io/gitpod/issues/8028#issuecomment-1069725188. Re-posting specs below for visibility, you can check out other [exploration designs](https://www.figma.com/file/3K7TIuko6ADxO0FY941huB/Display-warning-for-latest-IDE-versions-on-the-loading-screen-%238028?node-id=1111%3A259).

| Preparing | Running |
|-|-|
| <img width="1440" alt="Preparing" src="https://user-images.githubusercontent.com/120486/158705245-ee345d3d-1c03-4c20-9997-bc6ae61fb62b.png"> | <img width="1440" alt="Running" src="https://user-images.githubusercontent.com/120486/158705257-04b3bb54-8bac-4c18-a853-b81f69a20875.png"> |